test/test_encoding.ml

```diff
@@ -83,6 +83,13 @@ let tests = [
     next_option chars ok (assert_equal None);
     next_option chars ok (assert_equal None));
 
+  ("encoding.iso_8859_1" >:: fun _ ->
```

Comment:
Should this have been a test for `iso_8859_15` instead? It looks like an exact duplicate of the `iso_8859_1` test right now.